portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -101,13 +104,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String comment = getParameter(request, COMMENT, "");
+    String languageCode = getParameter(request, "lang", "");
     String user = getParameter(request, NAME, "");
     String userEmail = userService.getCurrentUser().getEmail();
     long timestamp = System.currentTimeMillis();
+
+    // Translate comment.
```

Comment:
nit: you probably don't need so many comments. Your use of variable names and the functions makes this "self-documenting." A person can pretty reasonably understand what is going on without needing a comment. Typically, we use comments for describing how functions and classes behave, to describe the why of complicated code, or to explain why we did something that is normally considered bad design.

You can read more here: https://google.github.io/styleguide/cppguide.html#Comments

Reply:
Great, thanks